integration_tests/src/main/python/udf_cudf_test.py

```diff
@@ -54,64 +59,68 @@ def _assert_cpu_gpu(cpu_func, gpu_func, cpu_conf={}, gpu_conf={}, is_sort=False)
         assert cpu_ret.sort() == gpu_ret.sort()
     else:
         assert cpu_ret == gpu_ret
-        
 
+
+# ======= Test Scalar =======
 @pandas_udf('int')
 def _plus_one_cpu_func(v: pd.Series) -> pd.Series:
     return v + 1
 
+
```

Comment:
nit remove extra newline

Reply:
Actually two lines between two functions is recommended by the IntelliJ IDE.